Sprint-1/exercises/random.js

```diff
@@ -1,9 +1,39 @@
 const minimum = 1;
 const maximum = 100;
 
-const num = Math.floor(Math.random() * (maximum - minimum + 1)) + minimum;
+// const num = Math.floor(Math.random() * (maximum - minimum + 1)) + minimum;
 
 // In this exercise, you will need to work out what num represents?
+/* Each time this code runs, num will represent a random integer between 1 and 100 */
+
 // Try breaking down the expression and using documentation to explain what it means
+/*
+
+- const minimum is setting the minimum value to 1 and const maximum is setting the maximum value to 100
+- Math.random() generates a random decimal number between 0(inclusive) and 1(exclusive) e.g. 0.354 or 0.928
+- Math.random() * (maximum - minimum + 1) this multiplication scales the random decimal to a range between 0 and 100 e.g. 0.5 * 100 = 50
+- (maximum - minimum + 1) is equal to 100 - 1 + 1 = 100
+- Math.floor() is taking the result of the multiplication and rounds it down to the nearest whole number, ensuring that the result is always an integer between 0 and 99 
+- The last part where we are adding the minimum to the result shifts the entire range up by 1, creating a range from 1 to 100 instead of the previous 0 to 99
+
+*/
+
```

Comment:
To test your understanding, how would you write an expression (without using any variable in the expression) that yields a random integer between -5 and 3 (including -5 and 3)?

Reply:
From the research, I gathered that I can used this: Math.floor(Math.random() * (3 - (-5) + 1)) + (-5)